src/lib.rs

```diff
@@ -158,12 +158,16 @@ Enable the `serde` feature to support the serde framework.
 
 ```toml
 [dependencies.cidr-utils]
-version = "*"
+version = "..."
```

Comment:
I prefer using star. Dot-dot-dot is not in correct format.

Reply:
removed this particular change from this PR